Jenkinsfile_nightly

```diff
@@ -24,7 +24,7 @@ properties([
   ])
 ])
 
-@Library("Infrastructure")
+@Library("Infrastructure@ccd-1257")
```

Comment:
Who's maintaining this `ccd-1257` branch? Are we 100% that there won't be any changes made there, potentially breaking our pipeline?

Reply:
CCD have added some new feature in infrastructure branch temporarily ,will revert to master once their testing is complete as it is all for testing, not final